src/EnergyPlus/General.cc

```diff
@@ -155,6 +155,168 @@ enum class RptKey
 constexpr std::array<std::string_view, static_cast<int>(RptKey::Num)> RptKeyNamesUC{
     "COSTINFO", "DXF", "DXF:WIREFRAME", "VRML", "VERTICES", "DETAILS", "DETAILSWITHVERTICES", "LINES"};
 
+// A second version that does not require a payload -- use lambdas
+void SolveRoot(EnergyPlusData &state,
```

Comment:
New non-template version of SolveRoot lives in the CC file.  Soon this will be the only one once we eliminate all the old Par usages.  I don't think I will try to do all of that in this one PR.